include/RAJA/exec-cuda/forallN_cuda.hxx

```diff
@@ -357,8 +357,11 @@ struct ForallN_Executor<ForallN_PolicyPair<CudaPolicy<CuARG0>, ISET0>,
                                 BODY body,
                                 CARGS const &... cargs) const
   {
-    cudaLauncherN<<<RAJA_CUDA_LAUNCH_PARAMS(dims.num_blocks, dims.num_threads)
-                 >>>(body, cargs...);
+    if (dims.num_blocks.x * dims.num_blocks.y * dims.num_blocks.z > 0 &&
```

Comment:
Can we have `numBlocks()` and `numThreads()` be functions that are re-used here, rather than spelling them out in the if-statement?